Kickstarter-iOS/ViewModels/UpdateViewModel.swift

```diff
@@ -5,10 +5,10 @@ import Prelude
 import ReactiveSwift
 import Result
 
-public struct UpdateData {
+private struct UpdateData {
```

Comment:
oh dang actually you don't even need to use this struct anymore since this `context` was for the unneeded events sry